scanner/engine/evaluate_worker.cpp

```diff
@@ -143,19 +94,21 @@ void* pre_evaluate_thread(void* arg) {
     std::vector<EvalWorkEntry> work_items;
     auto setup_start = now();
     for (size_t c = 0; c < work_entry.columns.size(); ++c) {
-      if (work_entry.column_types[c] == ColumnType::Video) {
+      if (work_entry.column_types[c] == ColumnType::Video &&
+          work_entry.video_encoding_type[media_col_idx] ==
```

Comment:
Should there be a `LOG_IF(FATAL) << "Unimplemented` if the encoding type is not H264?